### PR TITLE
build: improve localnet scripts and containers 

### PR DESCRIPTION
**Pull request by manu0466**
## Overview

This PR updates the Dockerfiles and scripts to run a localnet with all the contracts required for the liquid staking protocol. These updates account for the fact that the liquid staking contract now uses a single oracle contract and the newer version of the treasury contract.

closes: #XXXX

## What changes have been made in this PR?

- Refactored the scripts to use a set of shared params defined in the `params.sh` file
- Added a script to dowload all the required binaries
- Refactored the `init-stake-contract.sh` script to use  only a single oracle contract
- Updated the Dockerfiles to used the new scripts 

## Checklist

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation

### CODE REVIEW

File: scripts/testnet/local-osmosis-testnet-new.sh [hunk 1]
```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
+
 set -e
 
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+BINS_DIR=$SCRIPT_DIR/bins
+PATH=$BINS_DIR:$PATH
```
Comment (rustcandy) [inline scripts/testnet/local-osmosis-testnet-new.sh:7]: Should we export it? It doesn't seem to be used in this file

Comment (manu0466) [reply scripts/testnet/local-osmosis-testnet-new.sh:7]: Those variables are used to extend the `$PATH` variable, allowing the invocation of various binaries without specifying their path.

**Review by rustcandy — APPROVED** [review]
LGTM!

File: scripts/testnet/params.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+OSMOSIS_TRADER="osmo1lh0u9sug6qh922gjpal3frwtacaums4s7lkyl9"
+OSMOSIS_TRADER_ACCOUNT_INDEX=1
+
+# Celestia accounts
+CELESTIA_STAKER="celestia1sfhy3emrgp26wnzuu64p06kpkxd9phel74e0yx"
+CELESTIA_STAKER_ACCOUNT_INDEX=0
+CELESTIA_GRANTEE="celestia1lh0u9sug6qh922gjpal3frwtacaums4s8w5yn6"
+CELESTIA_GRANTEE_ACCOUNT_INDEX=1
+CELESTIA_REWARDS_COLLECTOR="celestia12rzczckgh8fqq533t0xqhqrrzdk76du3dxrx9q"
+CELESTIA_REWARDS_COLLECTOR_ACCOUNT_INDEX=2
```
[the remark this answers is not in the document]
Comment (manu0466) [reply scripts/testnet/params.sh:30]: Those don't need to be exported since this script is sourced when those params are neded

**Comment by RiccardoM** [comment]
@coderabbitai review

**Review by RiccardoM — APPROVED** [review]
utACK

**Pull request merged**